services/QuillLMS/app/controllers/teachers/units_controller.rb

```diff
@@ -131,6 +131,20 @@ def hide
     render json: {}
   end
 
+  def close
+    unit = Unit.find(params[:id])
+    unit.update(open: false)
+    ResetLessonCacheWorker.new.perform(current_user.id)
+    render json: {}
+  end
+
+  def reopen
```

Comment:
Super minor point, but this could also safely just be called `open`.  Theoretically you could imagine a future where some units start `closed`, and thus are not "re" opening.  It almost certainly doesn't matter, but I figured I'd flag it.